content/_data/upgrades/2-221-1.adoc

```diff
@@ -0,0 +1,76 @@
+==== Header and Breadcrumb Layout Improvements
+
+Instances with plugins that depend on specific details of the Jenkins layout (like the https://plugins.jenkins.io/simple-theme-plugin/[Simple Theme plugin]) may have user interface issues with this release.
+In the case of the Simple Them Plugin, users or theme maintainers may need to update CSS and JavaScript to match the new layout structure.
+
+==== Removed Agent Protocols
+
+Remoting changelogs for https://github.com/jenkinsci/remoting/releases/tag/remoting-4.2[4.2],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-4.0.1[4.0.1],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-4.0[4.0],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.40[3.40],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.39[3.39].
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.38[3.38], and
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.37[3.37]
+
+The Remoting library has been updated to resolve a security issue in remoting versions prior to 4.0.1 and to add support for the <<websocket-protocol,WebSocket protocol>>.
+The update removes outdated agent protocols that had previously been deprecated.
```

Comment:
```suggestion
The update removes outdated agent protocols that had previously been link:/blog/2017/08/11/remoting-update/[deprecated in Jenkins 2.75].
```

https://jenkins.io/blog/2017/08/11/remoting-update/ also provides an example of extended upgrade guidelines for Remoting